crates/query-engine/metadata/src/metadata/database.rs

```diff
@@ -41,6 +52,7 @@ pub struct ScalarType {
     pub aggregate_functions: BTreeMap<models::AggregateFunctionName, AggregateFunction>,
     pub comparison_operators: BTreeMap<models::ComparisonOperatorName, ComparisonOperator>,
     pub type_representation: Option<TypeRepresentation>,
+    pub full_type_representation: Option<TypeRepresentation>,
```

Comment:
What is the difference between `type_representation` and `full_type_representation`?

Reply:
Oh good catch meant to rip that out.